### ISSUE TITLE
Verify/augment external citations to 508 compliance

### ISSUE THREAD

**Issue by cmc333333** [18F/atf-eregs#261]
Hadiza was concerned that linking to external sources (which open in a new tab) require additional notification screen reader users. Confirm with our 508 colleagues how this should be handled.

**Comment by cmc333333** [comment]
@coreycaitlin shared a UCal reference: http://www.ucop.edu/electronic-accessibility/web-developers/inform-users-when-a-link-opens-in-a%20new-%20window.html

**Comment by cmc333333** [comment]
@nickbristow suggested adding a warning in the aria-description

**Comment by nickbristow** [comment]
@cmc333333 
`<a href="#" aria-label="Link opens in new window" target='_blank'>link</a>`
`<a href="#" title="Link opens in new window" target='_blank'>link</a>`
`<a href="#" target='_blank'>Link<span class="sr-only">Opens in new window</span></a>`
I like the first option

**Comment by cmc333333** [comment]
Merged, along with 18F/regulations-site#78
